assets/blocks/course-outline/module-block/module.scss

```diff
@@ -1,5 +1,29 @@
 $block: '.wp-block-sensei-lms-course-outline-module';
 
+body.theme-course {
```

Comment:
Is it necessary to include an element selector. Is the class specific enough?

Reply:
Also, I think it may be better to keep all theme-specific styles in the `3rd-party/css/<theme>` folder so that we don't load bloated CSS files for themes that don't need them.

Reply:
I've moved the styles to course specific stylesheets here https://github.com/Automattic/sensei/pull/7068/commits/351119595e9abc8adea277ec3abfad4cd60b162a.

The course-theme specific styles were only being loaded for learning mode before. I've stacked this PR on the variation stylesheet PR. The for the default variation of course theme, the title was looking a bit small, so I've changed it to be 2.75rem max instead of 2.5 only for default here https://github.com/Automattic/sensei/pull/7068/commits/7ef198dde39dfc1ea6a79ea7aac835f1ff5d8fbd.